cogs/card.py

```diff
@@ -189,9 +189,11 @@ async def convertmass(self, ctx: commands.Context, category: str):
         for card_id in user["cards"]:
             card = iufi.CardPool.get_card(card_id)
             if card:
-                if category == "notag" and not card.tag:
-                    pass
-                elif category != card.tier[1]:
+                if "notag" in categorys and not card.tag:
+                    if card.tier[1] not in categorys:
+                        continue
+                    
+                elif card.tier[1] not in categorys:
```

Comment:
so if I do "qcm notag" , it won't qc anything?

Reply:
> so if I do "qcm notag" , it won't qc anything?

umm you are right haha > . <